fileTemplates/Stateful Workflow.kt

```diff
@@ -1,35 +1,25 @@
-#set( $WorkflowName = "${NAME}Workflow" )
 package ${PACKAGE_NAME}
 
 import com.squareup.workflow1.Snapshot
 import com.squareup.workflow1.StatefulWorkflow
-import ${PACKAGE_NAME}.${WorkflowName}.Output
-import ${PACKAGE_NAME}.${WorkflowName}.Props
-import ${PACKAGE_NAME}.${WorkflowName}.Rendering
-import ${PACKAGE_NAME}.${WorkflowName}.State
 
 #parse("File Header.java")
-class ${WorkflowName} : StatefulWorkflow<Props, State, Output, Rendering>() {
-
-  data class Props
-  data class State
-  data class Output
-  data class Rendering
+object ${NAME} : StatefulWorkflow<${PROPS_TYPE}, ${STATE_TYPE}, ${OUTPUT_TYPE}, ${RENDERING_TYPE}>() {
 
   override fun initialState(
-    props: Props,
-    snapshot: Snapshot?
-  ): State = TODO("Initialize state")
+    props: ${PROPS_TYPE},
+  snapshot: Snapshot?
+  ): ${STATE_TYPE} = TODO("Initialize state")
 
   override fun render(
-    renderProps: Props,
-    renderState: State,
-    context: RenderContext
-  ): Rendering {
+    props: ${PROPS_TYPE},
+  state: ${STATE_TYPE},
```

Comment:
`renderProps` and `renderState` parameters are the new API and should be maintained.

Reply:
👍 will revert